static/handler_test.go

```diff
@@ -156,11 +158,159 @@ var _ = Describe("func FromPackages() (handler analysis)", func() {
 				},
 			))
 		})
+		When("a handler is a type alias", func() {
+			BeforeEach(func() {
+				// Set the GODEBUG environment variable to enable type alias
+				// support.
+				//
+				// TODO: Remove this setting once the package is
+				// migrated to Go 1.23 that generates `types.Alias` types by
+				// default.
+				os.Setenv("GODEBUG", "gotypesalias=1")
+			})
+
+			AfterEach(func() {
+				os.Setenv("GODEBUG", "")
```

Comment:
Maybe best set it back to whatever it was before?

Reply:
Sure, lemme change that.